src/Symfony/Bridge/Twig/Extension/TranslationExtension.php

```diff
@@ -95,7 +95,7 @@ public function trans($message, array $arguments = array(), $domain = null, $loc
 
     public function transchoice($message, $count, array $arguments = array(), $domain = null, $locale = null)
     {
-        return $this->translator->transChoice($message, $count, array_merge(array('%count%' => $count), $arguments), $domain, $locale);
+        return $this->translator->transChoice($message, $count, $arguments, $domain, $locale);
```

Comment:
You need to keep this in case the twig bridge is used with an older version of the translator.